src/mscorlib/src/System/TimeZoneInfo.cs

```diff
@@ -1900,7 +1900,7 @@ private TimeSpan GetDaylightSavingsEndOffsetFromUtc(TimeSpan baseUtcOffset, Adju
         }
 
 
-        static private Boolean CheckIsDst(DateTime startTime, DateTime time, DateTime endTime,bool ignoreYearAdjustment) {
+        static private Boolean CheckIsDst(DateTime startTime, DateTime time, DateTime endTime,bool ignoreYearAdjustment, bool noDaylightTransitionsRule) {
             Boolean isDst;
```

Comment:
what you think in passing the rule object instead of Boolean? may be in the future we need to use more stuff from the rule.